gtk/src/light/gtk-3.20/_tweaks.scss

```diff
@@ -355,6 +215,10 @@ button.titlebutton {
       @include draw_circle($selected_bg_color);
     }
   }
+
+  &.appmenu {
+    &, &:backdrop { @include button(undecorated); }
+  }
 }
 
 * { text-shadow: none; }
```

Comment:
Somehow this change adds back shadows to control buttons

instead of this
![image](https://user-images.githubusercontent.com/2883614/62865771-9694bd00-bd0f-11e9-9a2d-11fd6af0815a.png)

we have this again
![image](https://user-images.githubusercontent.com/2883614/62865798-ac09e700-bd0f-11e9-9e09-08772470519c.png)

Reply:
only in selection-mode

Reply:
True, thanks!
Prbly the weird wildcard doesn't work in deeper hierarchy

Reply:
Fixed now :)